src/matcha_ml/core/core.py

```diff
@@ -76,6 +83,41 @@ def get(
     return result
 
 
+def destroy(resources: List[Tuple[str, str]]) -> None:
+    """Destroy resources.
```

Comment:
A slightly more descriptive docstring here would be good

Reply:
Updated to match the cli `destroy --help` description.